src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -197,12 +201,15 @@ public function toolbarAction(Request $request, $token)
             // the profiler is not enabled
         }
 
+        $toolbarVersion = Kernel::VERSION_ID < 20800 ? self::TOOLBAR_VERSION_1 : self::TOOLBAR_VERSION_2;
```

Comment:
this does not make sense. You are choosing the toolbar version based on the HttpKernel component being installed, while it actually depends on the WebProfilerBundle version (templates and styles are in WebProfilerBundle). So when this code is used, it will always be version 2 (until the day we implement a third iteration)